gearcmd/worker.go

```diff
@@ -40,11 +40,17 @@ func (conf TaskConfig) Process(job baseworker.Job) ([]byte, error) {
 	}
 	if err != nil {
 		data["error_message"] = err.Error()
-		log.Printf(kayvee.FormatLog("gearcmd", "error", "END_WITH_ERROR", data))
+		data["type"] = "gauge"
```

Comment:
@nathanleiby & @azylman - I am ok with either (leaving gauge closer to the value for context, or moving it up to reduce repetition). Let me know what the preference is, and I can change it if necessary.